cmd/clean.go

```diff
@@ -44,12 +44,15 @@ func NewCleanCommand() *cobra.Command {
 func clean(cmd *cobra.Command, args []string) error {
 	aur := cmd.Flag("aur").Value.String() == "true"
 	dnf := cmd.Flag("dnf").Value.String() == "true"
+	apk := cmd.Flag("dnf").Value.String() == "true"
```

Comment:
```suggestion
	apk := cmd.Flag("apk").Value.String() == "true"
```
Shouldn't this be apk?